bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2262,6 +2263,19 @@ namespace {
       return BindCppObject( addr, (Cppyy::TCppType_t)Cppyy::GetScope( "TObject" ), kFALSE );
    }
 
+   //- Pretty printing with cling::PrintValue
+   PyObject *ClingPrintValue(ObjectProxy *self)
+   {
+      PyObject* cppname = PyObject_GetAttrString((PyObject*) self, "__cppname__");
+      std::string className = PyROOT_PyUnicode_AsString(cppname);
+      Py_XDECREF(cppname);
+
+      std::stringstream calcPrintValue;
+      calcPrintValue << "cling::printValue((" << className << "*)" << self->GetObject() << ");";
```

Comment:
What if the classname is a lambda - does this code then trigger a reinstall of Windows, or similar atrocities? (i.e. please make sure error cases are handled.)

Reply:
I've added the check but I still need to figure out how we should respond if the classname is not a Python string.